events/lifestyles/scholarship_lifestyle/learning_theology_events.txt

```diff
@@ -30,7 +30,15 @@ learning_theology.1301 = {
 	title = learning_theology.1301.t
 	desc = {
 		desc = learning_theology.1301.desc_opening
-		desc = learning_theology.1301.desc
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					like_christianity_religion_trigger = yes
+				}
+				desc = learning_theology.1301.desc_regula_pastoralis
```

Comment:
Are you sure about this? I'm fairly sure that "Regula Pastoralis" book don't exist in Warcraft universe

Reply:
Done in da079bb9bc61d6d11f582d96d663f708d0a357e9.

Reply:
Commented in 5aa124129da77dab1f3073f093add28e7f6f0d2e.